### PR TITLE
Template for epic release checklist

### LINKED ISSUE #11286: Checklist for epic features and improve DoD
Things to improve in our DoD docs https://sap.github.io/spartacus-docs/definition-of-done/ :
- add a security review 
- add an a11y review 
- emphasise briefly the main goals for a11y
   - I guess it's just a smooth user journey using the keyboard. But to be discussed with Miguel @developpeurweb and clarified what is a smooth user journey
- add cross-browser testing (and add our Test Matrix of devices and OS there)
- don't introduce breaking changes in a PR meant for a minor release

For features requiring a separate feature lib:
- review the architecture of the lib
- provide add-schematics to install the lib
- proof if it works for a customer (run our installation script /scripts/install/run.sh with `$BRANCH=<epic-branch-name>`)

For features requiring sample data (new or changed) or specific Commerce backend version:
- merge necessary data to the sampledata repo
- setup a new server with those sample data (and required backend version, if needed)
- perform acceptance test manually and e2e tests using the SPA epic branch against this backend server
- if tests passed, then deploy the sample data (and backend version) in our CI test server
- only then can merge the feature to develop

For big features:
- request a threat modelling with Michael R.
- request an a11y audit with Miguel

### PR DESCRIPTION
**Pull request by hackergil**
Fixes #11286

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/epic_release.md
Comment (ChristophHi) [inline .github/ISSUE_TEMPLATE/epic_release.md]: Should we always start with @giancorderoortiz here? For our second increment, we are trying to organize a session with the central commerce team (Michael Riedel) like we did for the first iteration.

File: .github/ISSUE_TEMPLATE/epic_release.md
Comment (ChristophHi) [inline .github/ISSUE_TEMPLATE/epic_release.md:38]: - What if we have sample data for CI purposes that we don't want to share with customers (CPQ endpoints e.g.). For CPQ, our approach at the moment is to provide an extra script that will be deployed on top of commerce and sample data

File: .github/ISSUE_TEMPLATE/epic_release.md
Comment (ChristophHi) [inline .github/ISSUE_TEMPLATE/epic_release.md:22]: Maybe also add general coding best practises, the things @Platonn made the product configurator team aware of: (avoid 'public' explictly, no 'private' methods, avoid un-needed subscriptions on components but use the reactive pattern as far as possible...)

File: .github/ISSUE_TEMPLATE/epic_release.md
Comment (ChristophHi) [inline .github/ISSUE_TEMPLATE/epic_release.md:36]: With this checklist we are not really covering future scenarios that require modules from the SAP integration package (like we are discussing with GC at the moment. Commerce core, based on recipe 'cx' is no longer sufficient). It's only mentioned implicitly by 'Epic has sufficient end to end (cypress) test coverage'
 
Maybe we should have a section
```
## CI
- Ensure that we find a common commerce setup that can serve the entire SPA scope, including the new feature.

```

File: .github/ISSUE_TEMPLATE/epic_release.md
Comment (hackergil) [reply .github/ISSUE_TEMPLATE/epic_release.md:22]: Hi @ChristophHi . That is encompassed on the [coding guidelines](https://sap.github.io/spartacus-docs/coding-guidelines/) page, which is referenced on the DoD page.

File: .github/ISSUE_TEMPLATE/epic_release.md
Comment (hackergil) [reply .github/ISSUE_TEMPLATE/epic_release.md]: Yes, I think we should rely on teams rather than people. Thanks for the feedback.

File: .github/ISSUE_TEMPLATE/epic_release.md
Comment (hackergil) [reply .github/ISSUE_TEMPLATE/epic_release.md:36]: From the architects, out of the box `cx` recipe should satisfy this. Which is what we've been doing until now.

File: .github/ISSUE_TEMPLATE/epic_release.md
Comment (hackergil) [reply .github/ISSUE_TEMPLATE/epic_release.md:38]: Noted. We assume that all of the sample data is public and free to be shared.

**Comment by hackergil** [comment]
@Platonn If there's no more feedback on this, can you approve?

File: .github/ISSUE_TEMPLATE/epic_release.md
Comment (ChristophHi) [reply .github/ISSUE_TEMPLATE/epic_release.md:36]: No, cx recipe does not cover CPQ support. That's why we are having a discussion with @giancorderoortiz around setting up commerce for this flavor.

CPQ support _would_ be covered by cx-integrations, but other integration scenarios are not part of cx-integrations yet-> I fear we need to have these considerations as an explicit step

**Review by plabadie — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/epic_release.md
Comment (hackergil) [reply .github/ISSUE_TEMPLATE/epic_release.md:36]: Ok, thanks for letting me (us) know. Would you mind adding them after we merge this? Also, can you let us know if they are documented? Thanks.

**Pull request merged**
